components/menu/stories/menu.stories.js

```diff
@@ -375,9 +402,134 @@ Collapsible.args = {
 	],
 };
 
-// story used in Picker component
+// ********* DOCS ONLY ********* //
+/**
+ * A menu item should always have a label that clearly describes the action or option that it represents. A menu item
+ * can also display a related value in the value area. Examples of values include the selected option from a submenu, a
+ * keyboard shortcut for the action, or other content that clarifies the menu item. When necessary, menu items may also
+ * display an icon and additional description text.
+ *
+ * Menu sizes should correspond to the size of the menu trigger component (such as an
+ * [action button](?path=/docs/components-action-button--docs)). Similarly, any components displayed inside a menu item
+ * (such as a [switch](?path=/docs/components-switch--docs)) must also be of the same size.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.storyName = "Default";
+Sizing.tags = ["!dev"];
+Sizing.args = {
+	items: [
+		{
+			idx: 1,
+			label: "Menu item",
+		},
+		{
+			idx: 2,
+			label: "Menu item with icon",
```

Comment:
Do we like the menu being displayed like this with each item having some different kind of content (one with an icon, one with value, one with description)? I was trying to avoid needing so many stories, but can add more to display the content differently if this feels confusing.

Reply:
I think another item with both icon and description is a good idea!

If the story name changes to "Default with sizing", it'll show up after "Collapsible" instead of being the first story because of how Storybook alphabetizes stories. I think I kind of prefer keeping it as "Default" for that reason, but what do you think?

Reply:
👍 I think keeping as "Default" is fine then.